static/js/directives/report-image.js

```diff
@@ -0,0 +1,16 @@
+angular.module('inboxDirectives').directive('reportImage',
+  function(DB) {
+    return {
+      template: '<div class="loader"></div>',
+      link: function(scope, element, attr) {
+        DB().getAttachment(attr.report, attr.path, { binary:false })
+          .then(function(blob) {
+            var reader = new FileReader();
+            reader.readAsDataURL(blob);
+            reader.onloadend = function() {
+              $(element).replaceWith('<img class="report-image" src="data:' + blob.type + ';base64:' + reader.result + '"/>');
+            };
```

Comment:
We have similar but different code in Enketo.replaceJavarosaMediaWithLoaders - it uses URL.createObjectURL instead of a FileReader. I'm wondering what the right way is? If they're the same can we pull out a server to get the attachment and return the HTML?